style.css

```diff
@@ -91,3 +81,193 @@ h1 span {
 .page-links li {
   margin-right: 19px;
 }
+
+.my-projects {
+  padding: 15px;
+  margin: 20px;
+}
+
+.down-arrow {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 15px;
+}
+
+.title {
+  display: flex;
+  justify-content: center;
+  margin: 10px;
+}
+
+.line {
+  display: flex;
+  justify-content: center;
+  margin: 20px;
+}
+
+.project-container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 20px;
+}
+
+.single-project {
+  height: 474px;
+  width: 327px;
+  border-radius: 8px;
+  border: 2px solid #ebf0ee;
+  margin: 5px auto;
+}
```

Comment:
- Please ensure that your cards have the exact margins to the right and left of your screen. I see you have applied the `margin: 5px auto` but the cards are not still at the center. Please refer to the provided [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).
### See your design
![your-design](https://user-images.githubusercontent.com/58692590/148635553-e3257070-54f2-492b-946a-f95b87481ba0.png)
### Figma Template
![figma](https://user-images.githubusercontent.com/58692590/148635587-35e7fb42-5c44-4054-a7fb-2b06c6857706.png)

Reply:
Hello @iLynette  You seem to have made matters somehow worse in fixing this. The margins are worse than before.